static/app/utils/discover/urls.tsx

```diff
@@ -31,7 +31,7 @@ export function eventDetailsRoute({
   eventSlug: string;
   orgSlug: string;
 }): string {
-  return `/organizations/${orgSlug}/discover/${eventSlug}/`;
+  return normalizeUrl(`/organizations/${orgSlug}/discover/${eventSlug}/`);
```

Comment:
Why do we need this now?

Reply:
It's so that it can eventually work with self-hosted, we are encouraged to normalize all urls